2022-03-07 18:11:18,148 web.py[line:1789] ERROR Uncaught exception POST /users/login (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:8000', method='POST', uri='/users/login', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/Users/qiangchen/Desktop/Labs/Sao/venv/lib/python3.9/site-packages/tornado/web.py", line 1702, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/qiangchen/Desktop/Labs/Sao/views/users/users_views.py", line 27, in post
    ex_user = User.query.filter(phone=phone).first()
TypeError: filter() got an unexpected keyword argument 'phone'
2022-03-07 18:11:18,157 web.py[line:2239] ERROR 500 POST /users/login (127.0.0.1) 20.50ms
